modules/internal/ChapelArray.chpl

```diff
@@ -2027,7 +2030,28 @@ module ChapelArray {
     proc tail(): this._value.eltType {
       return this[this.domain.high];
     }
-
```

Comment:
There is a comment earlier in this file that needs updating:

       The current implementation reallocates the array every time the domain
       is modified.  This could be improved with a size doubling/halving
       strategy.